client/asset/btc/btc.go

```diff
@@ -2573,6 +2573,34 @@ func (btc *ExchangeWallet) PayFee(address string, regFee, feeRateSuggestion uint
 	return newOutput(txHash, vout, sent), nil
 }
 
+// feeRateWithOracleFallback attempts to retrieve a fee rate from the wallet
+// but falls back to the oracle function if it fails.
+func (btc *ExchangeWallet) feeRateWithOracleFallback(confTarget uint64, oracle func() uint64) uint64 {
+	feeRate, err := btc.estimateFee(btc.node, confTarget)
+	if err == nil {
+		btc.log.Tracef("Obtained local estimate for %d-conf fee rate, %d", confTarget, feeRate)
+		return feeRate
+	}
+	feeSuggestion := oracle()
+	if feeSuggestion > 0 && feeSuggestion < btc.fallbackFeeRate && feeSuggestion < btc.feeRateLimit {
```

Comment:
Why is `feeSuggestion < btc.fallbackFeeRate` a requirement? Not saying it's wrong, but asking for my understanding. If the network is busy it could be higher. As long as it isn't over `btc.feeRateLimit` I would think it's fine.

Reply:
Actually I'm not sure why either, but this is the logic in the existing code (`feeRateWithFallback`) as well. Does anyone else know why? @chappjc @buck54321

Reply:
Hmm, not really.  I would think that `feeRateLimit` would be adequate to compare.